### DISCUSSION TITLE
closed

### DISCUSSION THREAD

**Discussion by EliasKramer** [git-lfs/git-lfs#5046] [Q&A] [answered]

**Comment by bk2204** [comment, marked as answer]
Hey,

If these files were already in your repository before you ran `git lfs install`, running `git lfs track` isn't sufficient, since it doesn't rewrite history, and GitHub doesn't allow large files anywhere in the history.  In that case, you'd need to run `git lfs migrate import --everything --include "*.txt"` to rewrite the entire history, but note that all of your object IDs for your commits will change.
